src/Screens.js

```diff
@@ -1,6 +1,12 @@
 import { Navigation } from 'react-native-navigation'
+import { Provider } from 'react-redux'
+
 import SignInScreen from './Containers/SignInScreen'
+import RideList from './Containers/RideList'
+import TestScreen from './Containers/TestScreen'
 
-export default function () {
-  Navigation.registerComponent('example.Types', () => SignInScreen)
+export default function (store) {
+  Navigation.registerComponent('carona.signIn', () => SignInScreen, store, Provider)
```

Comment:
`() => SignInScreen, store, Provider` is a wierd syntax, but it's the correct syntax
[registerComponent API](https://wix.github.io/react-native-navigation/#/top-level-api?id=registercomponentscreenid-generator-store-undefined-provider-undefined)